packages/webapp/src/containers/Finances/OtherExpense/index.jsx

```diff
@@ -129,7 +129,7 @@ class OtherExpense extends Component {
         let amount = parseFloat(e.value);
         subTotal += amount;
         detailedHistory.push({
-          date: moment(e.expense_date),
+          date: moment(e.expense_date).toDate().toLocaleDateString(),
```

Comment:
I think it is better to do this on line 181. (That is what `Cell` is for)
`Cell: (d) => <span>{d.value.toDate().toLocaleDateString()}</span>`

Is this `mm/dd/yyyy` the correct format in North America?

Reply:
North America uses `mm/dd/yyyy` date format. We can check it here https://iso.mit.edu/americanisms/date-format-in-the-united-states/

Reply:
👍 I will never get used to this format 😵‍💫